src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,47 @@
+package pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.FindBy;
+import java.time.Duration;
+public class HomePage extends BasePage {
+    @FindBy (css = "[img.avatar]")
+    WebElement userAvatarIcon;
+    @FindBy (xpath = "//*[@id=\"playlists\"]/ul/li[3]/a")
+    WebElement thirdPlaylist;
+    @FindBy  (xpath = "//*[@id=\"playlists\"]/ul/li[3]/nav/ul/li[1]")
+    WebElement editBtn;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```